public/widget.js

```diff
@@ -0,0 +1,222 @@
+(function () {
+  const host = 'https://li.finance'
```

Comment:
Do we want to make this configurable as well to allow e.g. local testing or staging?

Reply:
The same issue as with TS we don't have access to env variables here. If you have any suggestions, I would be happy to change this. 😄

Reply:
Hmm yeah I get it. Would it make sense to pass it as a configuration parameter? I guess "users" should always use the production version so it doesn't make too much sense?
Let's just leave it for now I think.

Reply:
Yep, agree. Let's move forward.